docs/components/navbar.md

```diff
@@ -178,6 +178,14 @@ Navbars can be statically placed (their default behavior), or fixed to the top o
 </nav>
 {% endexample %}
 
+When using full width, statically placed navbars you might want to remove the rounded corners, this can be done by including `.navbar-static-top`.
```

Comment:
Good point however taking a look at the code (https://github.com/twbs/bootstrap/blob/v4-dev/scss/_navbar.scss#L23-L29) it seems that it doesn't modifies any position related CSS, unlike `.navbar-fixed-top`  and `.navbar-fixed-bottom` (https://github.com/twbs/bootstrap/blob/v4-dev/scss/_navbar.scss#L32-L52).

<img width="1280" alt="screen shot 2015-08-22 at 20 20 21" src="https://cloud.githubusercontent.com/assets/5364897/9425280/4062ed52-490b-11e5-9299-02da5f6a2070.png">